package on2018_01.on2018_01_02_HourRank_25.The_Strange_Function;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2018_01/on2018_01_02_HourRank_25/The_Strange_Function/The Strange Function.task"))
			Assert.fail();
	}
}
